specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/stable/2018-09-01/containerregistry_build.json

```diff
@@ -0,0 +1,1807 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2018-09-01",
+    "title": "ContainerRegistryManagementClient"
+  },
+  "host": "management.azure.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ContainerRegistry/registries/{registryName}/scheduleRun": {
+      "post": {
+        "tags": [
+          "Registries"
+        ],
+        "description": "Schedules a new run based on the request parameters and add it to the run queue.",
+        "operationId": "Registries_ScheduleRun",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/RegistryNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "runRequest",
+            "in": "body",
+            "description": "The parameters of a run that needs to scheduled.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/RunRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request was successful; the request was well-formed and received properly.",
+            "schema": {
+              "$ref": "#/definitions/Run"
+            }
+          },
+          "202": {
+            "description": "The request was successfully accepted; the operation will complete asynchronously."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed. If the registry/run doesn't exist, 404 (Not found) is returned. If any of the input parameters is wrong, 400(Bad Request) is returned."
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Registries_ScheduleRun": {
+            "$ref": "./examples/RegistriesScheduleRun.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ContainerRegistry/registries/{registryName}/getBuildSourceUploadUrl": {
```

Comment:
Is this API ever expected to be consumed from an ARM template? If so, @ravbhatnagar, is it still true that the name should be list*** (regardless of how many items are actually returned)?

Reply:
Yes, you are correct. It cannot be called from ARM template if it is not prefixed with list. There is no use case for this API to be called from template

Reply:
In general, I've often found that there are scenarios for consuming pretty much any computed value from within a template. So, my default recommendation is to take an "assume it will be consumed from within a template unless proven otherwise" approach. But this is a general statement. If you truly don't have any scenarios where one would, say, provision a virtual machine that needs the access the value (or that user assigned identities would enable the access from the resource itself), then this is all good.

Reply:
I renamed it to prefix with list so that it can be invoked from template.